docs/diagrams/MainLogicSequenceDiagram.puml

```diff
@@ -19,6 +19,7 @@ end box
 
 autoactivate on
 
+mainFrame sd Logic for MediVault
```

Comment:
I dont think this is necessary because this is the main logic and nothing is referencing it.

Reply:
Ok, I've updated it